sympy/multipledispatch/core.py

```diff
@@ -81,5 +81,5 @@ def ismethod(func):
         signature = inspect.signature(func)
         return signature.parameters.get('self', None) is not None
     else:
-        spec = inspect.getargspec(func)
+        spec = inspect.getfullargspec(func)
```

Comment:
`sympy/multipledispatch` is a clone of https://github.com/mrocklin/multipledispatch

Maybe you should open a PR over there.

Reply:
@Upabjojr , Thanks.